.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Run tests
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Think `main` is currently still called `master` but let's fix that.

Reply:
Oh yes, that explains why the workflow didn't run! Thanks for doing the rename :smile: